package.json

```diff
@@ -59,14 +67,18 @@
   "devDependencies": {
     "babel-cli": "^6.18.0",
     "babel-core": "^6.21.0",
+    "babel-eslint": "^7.1.1",
```

Comment:
Not sure `babel-eslint` is mandatory for `standardjs`, but it may be mandatory for `flow`. That is weird.